backend/src/main/java/org/cryptomator/hub/api/VaultResource.java

```diff
@@ -307,12 +312,12 @@ public Response create(@PathParam("vaultId") @Pattern(regexp = ValidationUtil.ID
 		}
 	}
 
-	public record VaultDto(@JsonProperty("id") @NotBlank @Pattern(regexp = ValidationUtil.ID_PATTERN) String id,
-						   @JsonProperty("name") @NotBlank @Pattern(regexp = ValidationUtil.NAME_PATTERN) String name,
-						   @JsonProperty("description") @Pattern(regexp = "(?U)[-\\p{Alnum}\\h_.~!%&'()*+;=:/?#@]*") String description,
+	public record VaultDto(@JsonProperty("id") @IsUUID String id,
+						   @JsonProperty("name") @AlmostSafeText String name,
+						   @JsonProperty("description") @Pattern(regexp = "^[&*<>\"]") String description,
```

Comment:
`^` at this position means "start of string/line", you need to put it inside the `[^...]` .

Reply:
Fixed in https://github.com/cryptomator/hub/pull/154/commits/7960c3fa1ea1cdad4ac7d2172a68454b398227f8